lib/screens/create_region_screens/interactor/viewmodels/create_region_state.dart

```diff
@@ -97,8 +102,8 @@ class CreateRegionState extends Equatable {
       isRegionDescriptionNextButtonEnable: false,
       isUploadImageNextButtonEnable: false,
       pictureBoxState: PictureBoxState.pickImage,
-      isRegionIdNextButtonEnable: false,
       regionId: "",
+      regionIdAuthenticationState: AuthenticationState.loading,
```

Comment:
is initial state loading?

Reply:
The first time is always gonna be on loading because it needs to validate the Id generated with the values of pervious screen